Sources/KIF/Additions/UIView-KIFAdditions.m

```diff
@@ -692,6 +701,7 @@ - (void)dragPointsAlongPaths:(NSArray<NSArray<NSValue *> *> *)arrayOfPaths {
                 point = [self convertPoint:point fromView:self.window];
                 UITouch *touch = [[UITouch alloc] initAtPoint:point inView:self];
                 [touch setPhaseAndUpdateTimestamp:UITouchPhaseBegan];
+                [touch setIsFromEdge:isFromEdge];
```

Comment:
I wonder if we should alternatively come up with a heuristic here for what constitutes an edge swipe and do it implicitly. Being explicit seems reasonable as well though.

Reply:
I think with this we can make an API specifically for swiping at an edge, similarly to like tapping on a button or status bar instead of using x,y coordinatotes. As well as implicitly doing it from the edge if we can detect it being around where it should be. I think that'd be in line with other APIs of KIf

Reply:
I thought about doing it implicitly like
```
if (startPoint.x < 1 || startPoint.x > screen.size.width - 1) {
    [touch setIsFromEdge:isFromEdge];
}
```
But this could be a breaking change for users who utilize the `dragFromPoint` method without intending to trigger a screen edge pan gesture.

Reply:
I have tried to implement an alternative way based on the suggestions, which one do you prefer?